src/crypto-providers/util.ts

```diff
@@ -379,42 +374,38 @@ const _packBaseAddress = (
   return {
     address,
     addressType: getAddressType(address),
-    paymentPath: paymentPathFile.path,
-    stakePath: stakePathFile.path,
+    paymentPath: paymentSigningFile.path,
+    stakePath: stakeSigningFile.path,
   }
 }
 
 const _packEnterpriseAddress = (
-  hwsData: HwSigningData, network: Network,
+  paymentSigningFile: HwSigningData, network: Network,
 ): _AddressParameters => {
-  const { pubKey: paymentPubKey } = destructXPubKeyCborHex(hwsData.cborXPubKeyHex)
+  const { pubKey: paymentPubKey } = destructXPubKeyCborHex(paymentSigningFile.cborXPubKeyHex)
   const address: Buffer = packEnterpriseAddress(
     getPubKeyBlake2b224Hash(paymentPubKey),
     network.networkId,
   )
   return {
     address,
     addressType: getAddressType(address),
-    paymentPath: hwsData.path,
+    paymentPath: paymentSigningFile.path,
   }
 }
 
 const _packRewardAddress = (
-  hwsData: HwSigningData[], network: Network,
+  stakeSigningFile: HwSigningData, network: Network,
 ): _AddressParameters | null => {
```

Comment:
I think we should not include `null` here, please check for all the other similar functions.

Reply:
Fixed